development/image-builder/main.go

```diff
@@ -209,6 +210,17 @@ func runBuildJob(o options, vs Variants, envs map[string]string) error {
 		buildArgs = envs
 	}
 
+	if o.buildArgs != nil && len(o.buildArgs) != 0 {
+		parsedBuildArgs, err := getTags(pr, sha, append(o.buildArgs, o.TagTemplate))
```

Comment:
You should not use `getTags()` here. You don't need `Tagger` to be invoked for those values if you treat `sets.Tags{}` as Named Strings list. Just access the fields directly.